common/vsprops/preBuild.cmd

```diff
@@ -55,6 +55,16 @@ if %ERRORLEVEL% NEQ 0 (
     echo #define SVN_REV 0ll >> "%CD%\svnrev.h"
     echo #define GIT_REV "" >> "%CD%\svnrev.h"
     echo #define GIT_TAG "%GIT_TAG%" >> "%CD%\svnrev.h"
+
+    echo %GIT_TAG%|FINDSTR /R "^v[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*$" > NUL
```

Comment:
`+` wasn't supported in findstr [MS Docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/findstr#remarks)

I don't know about `\d`, tellow's regex was already using `[0-9]`, but I doubt findstr supports it

Reply:
My regex used `[0-9]` because I couldn't get cmake to recognize `\d`
Can't find an actual reference on cmake's regex so I don't know if I was just messing it up or if it actually isn't supported though